controllers/twophase/types.go

```diff
@@ -83,11 +83,16 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 
 	scheduler := chaos.GetScheduler()
-	if duration == nil || scheduler == nil {
-		r.Log.Info("scheduler and duration should be defined currently")
+	if scheduler == nil {
+		r.Log.Info("scheduler should be defined currently")
```

Comment:
For `container-kill` and `pod-kill`, this is a normal log, for the other action, this is an error.

Reply:
Not actually, the twophase controller let the `duration` become 0 second when it is omitted which would take effect nearly 10 seconds as default resync interval.
And further more, we can validate the chaos Spec in their own chaos kind controller or validation webhook when created or updated.

Reply:
Sorry, I misunderstand this check. I think we need to return `ctrl.Result{RequeueAfter: recover-duration}` when to set recover time.